MC/dpgsim.sh

```diff
@@ -429,6 +432,10 @@ while [ ! -z "$1" ]; do
 	CONFIG_KEEPTRACKREFSFRACTION="$1"
 	export CONFIG_KEEPTRACKREFSFRACTION
         shift
+    elif [ "$option" = "--signalFilteringFraction" ]; then
```

Comment:
Isn't it worth to consider this option only in case the signal filtering is on?
This check is somehow later, actually. But maybe one could do it as first thing.

Reply:
Hi @chiarazampolli 
good point, I can add that.

Reply:
Hi @chiarazampolli,

done, see below.

Cheers,
Michael

Reply:
Thanks!